log/gps/gps_mgl_enigma_waypoint.ksy

```diff
@@ -0,0 +1,134 @@
+meta:
+  id: gps_mgl_enigma_waypoint
+  title: MGL Avionics Enigma Waypoint format
+  application: Enigma software
+  file-extension:
+    - ewd # WAYPOINT.EWD usually
+    - rte # MYROUTE.RTE usually
+  license: Unlicense
+  endian: le
+  encoding: ascii
+
+-license-header: | # the conditions contradict each other and seem to be unenforceable
```

Comment:
I think this would fit better under `meta`, because that's where the normal `license` attribute goes.

Reply:
Fixed, thanks.